packages/rocketchat-ui/client/components/header/header.js

```diff
@@ -57,8 +57,21 @@ Template.header.helpers({
 
 	roomTopic() {
 		const roomData = Session.get(`roomData${ this._id }`);
-		if (!roomData) { return ''; }
-		return roomData.topic;
+		if (!roomData || !roomData.topic) { return ''; }
+
+		let roomTopic = RocketChat.Markdown.parse(roomData.topic);
+
+		// &#39; to apostrophe (') for emojis such as :')
+		roomTopic = roomTopic.replace(/&#39;/g, '\'');
+
+		Object.keys(RocketChat.emoji.packages).forEach((emojiPackage) => {
+			roomTopic = RocketChat.emoji.packages[emojiPackage].render(roomTopic);
+		});
+
+		// apostrophe (') back to &#39;
+		roomTopic = roomTopic.replace(/\'/g, '&#39;');
+
+		return roomTopic;
```

Comment:
I'd like this to be reviewed for possible XSS injection.

Reply:
LGTM